backend/database/user.go

```diff
@@ -43,12 +44,20 @@ func (mDb MongoDatabase) GetUserByEmail(ctx context.Context, email string) (*Use
 	return user, nil
 }
 
+func (mDb MongoDatabase) SetUserRole(ctx context.Context, userID primitive.ObjectID, role string) error {
```

Comment:
```suggestion
func (mDb MongoDatabase) SetUserRole(ctx context.Context, userID ObjectID, role string) error {
```

Reply:
I'd suggest to change the userID type to ObjectID to keep it consistent with the rest of the code. Also, don’t forget to remove the unused import afterward (a comment I left above).